src/tests/integration/test-data/deployment-code.expected

```diff
@@ -0,0 +1,7 @@
+pragma solidity ^0.8.13;
+
+import { Vm } from "forge-std/Vm.sol";
+
+contract DeploymentSummary1Code {
```

Comment:
Where does the `1` come from?

Reply:
I wanted to have some way of showing it was an example and not built-in, but I see why that's confusing, I've renamed it to simply `DeploymentSummary`.